app/views/tag/show.html.erb

```diff
@@ -34,7 +34,7 @@
   background-size: cover;
   background-color: #FFF;
 }
-.fdb-block, .rounded {
+.rounded {
```

Comment:
Ah, i'm a little worried what happens when this has a background image now. Could you go to the corresponding wiki page (or create one if you need, where `/tag/tagname` corresponds to `/wiki/tagname`), and see if a tall image then goes out of the frame? If so, we may need to think of another way around this somehow...

Thanks for catching this issue though!!!!

Reply:
@jywarren I am sorry I am facing connection issues in my localhost development probably caused by rvm or redis...  I think it is causing other problems for me to make a first post and bypass the moderation system. Is there any place I can address my issue?

Reply:
Hi @manchere, do you have a specific error that you are getting?  we could try and debug

Reply:
@cesswairimu I would love to, thank you.

Reply:
feel free to post the error you are getting here or browse through #3840 you may find a fix for it over there. Thanks